src/Compilers/CSharp/Portable/Symbols/Source/LambdaSymbol.cs

```diff
@@ -188,12 +148,6 @@ public override TypeWithAnnotations ReturnTypeWithAnnotations
             get { return _returnType; }
         }
 
-        public override FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations => FlowAnalysisAnnotations.None;
```

Comment:
Please include a test for nullable return type and nullability attributes (Maybe/NotNull/NotNullIfNotNull). Do nullability checks on returned values work properly?

Reply:
> Do nullability checks on returned values work properly?

Added tests with PROTOTYPE comment.